files-community/Files
# Feature: Added a new option to remove checkboxes under folder settings

![showcheckbox](https://user-images.githubusercontent.com/42214331/235482958-daf428fc-fd9d-4a93-921b-30eb0d54897d.png)
This changes adds a new feature "show checkboxes", turn it off to remove checkboxes completely   

**Validation**
How did you test these changes?
 1. Go to settings 
 2. Click on Folders 
 3. Under Display a new setting "Show checkboxes" should be automatic on
 4. Turn it off and  if **show-checkboxes-when-selecting-items** is on it will automatic turn it off too
 5. Now you shouldn't see any checkboxes
